drivers/timer/nrf_rtc_timer.c

```diff
@@ -452,7 +452,15 @@ void _timer_idle_exit(void)
  * this example. The ISR will then announce the number of sys ticks it was
  * delayed (2), and schedule the next sys tick (5) at 500.
  */
+#if (defined(CONFIG_ZTEST) && (!defined(CONFIG_GEN_ISR_TABLES)))
```

Comment:
is there a code size implication from removing 'static' so we don't have this ugly ifdef?

Reply:
No, there is no size implication, IMO; if the function is not used, it is not linked.

The main reason for the #ifdef is to expose as little API as necessary; (de-scoping this function prevents from having it accidentally used in the code, something that can possibly lead to implementation bugs).

Reply:
I understand, but it feels wrong to me to change the code just for test like this.

Reply:
agree with @galak

Reply:
@nashif  @galak  thanks for your input; I have no strong opinion, so I will remove this #ifdef block, then.